content/GOVERNANCE.md

```diff
@@ -27,9 +27,9 @@ handled on a case-by-case basis.
 - [Valentin Sulzer](https://sites.google.com/view/valentinsulzer) - Ionworks
 - [Martin Robinson](https://www.sabsr3.ox.ac.uk/people/dr-martin-robinson) - University of Oxford
 
-## Previous Members
+## Retired Members
 
-- [Saransh Chopra](https://saransh-cpp.github.io)
+- [Saransh Chopra](https://saransh-cpp.github.io) (2022-2025) - University College London
```

Comment:
Very nice how my tenure came out to be three years when I stepped down, aligns with the new rules.